homeassistant/components/google_generative_ai_conversation/strings.json

```diff
@@ -43,6 +43,9 @@
           "prompt": "Instruct how the LLM should respond. This can be a template."
```

Comment:
Can you add a data_description for enable_google_search_tool mentioning the "No control" requirement?